index.bs

```diff
@@ -319,6 +320,26 @@ To determine whether an |attribute| and |element| <dfn>attribute-match</dfn> an
 4. if |matches| contains the string "*", return true.
 5. return false.
 
+Some HTML elements require special treatment in a way that can't be easily
+expressed in terms of configuration options or other algorithms. The following
+algorithm collects these in one place. To <dfn>handle funky elements</dfn>,
+run these steps:
+
+1. if |element|'s [=element interface=] is {{HTMLTemplateElement}}:
+  2. Run the steps of the [=sanitize document fragment=] algorithm on
+     |element|'s |content| attribute, and replace |element|'s |content|
+     attribute with the result.
+  3. Drop all child nodes of |element|.
+2. if |element|'s [=element interface=] has a {{HTMLHyperlinkElementUtils}} mixin:
+  1. if |element|'s `protocol` property is "javascript:", then remove the
+     `href` attribute from |element|.
+3. if |element|'s [=element interface=] is {{HTMLFormElement}}:
+  1. if |element|'s `action` attribute is a [[URL]] with `javascript:`
+     protocol, remove the `action` attribute.
+4. if |element|'s [=element interface=] is {{HTMLInputElement}}:
```

Comment:
nit: formaction attribute also exists on HTMLButtonElement

Reply:
Yes, thank you. Added.